### ISSUE TITLE
Ffmpeg return Broken pipe on Windows platform

### ISSUE THREAD

**Issue by cytsai1008** [Pycord-Development/pycord#1501]
### Summary

Ffmpeg return Broken pipe on Windows platform

### Reproduction Steps

1. Bot join a voice channel
2. Let bot play something (ex. mp3)

### Minimal Reproducible Code

```python
voice_file = discord.FFmpegOpusAudio(f"tts_temp/{guild_id}.mp3")
ctx.voice_client.play(voice_file)
```

### Expected Results

Bot plays music
stay in vc and waiting for next play command

### Actual Results

Played nothing and sudden disconnected
Below error printed after I killed the bot
```
av_interleaved_write_frame(): Broken pipe
Error writing trailer of pipe:1: Broken pipe
Error closing file pipe:1: Broken pipe
```

### Intents

discord.Intents.all()

### System Information

- Python v3.10.5-final
- py-cord v2.0.0-final
- aiohttp v3.8.1
- system info: Windows 10 10.0.19044

BTW, my ffmpeg version: `ffmpeg version 5.0.1-essentials_build-www.gyan.dev`

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

I've tested it works perfectly on Ubuntu 22.04 and after some Google I think it was because of Windows or powershell's pipe was different with Linux
And it doesn't happened on py-cord v1.7.3
And `discord.FFmpegPCMAudio(f"tts_temp/{guild_id}.mp3")` also return the same problem

**Comment by davidhozic** [comment]
Same happens to me

**Comment by Lulalaby** [comment]
Status Update please 
@Pycord-Development/maintainers

**Comment by Middledot** [comment]
When testing, I didn't get an error but no audio would play. This seems to have been fixed with #1955 (88cc1878860f93a1b71397350c103fd4bf8f195e) in v2.5
